### DISCUSSION TITLE
Request for Assistance with Default Experiment Settings in pythonfmu

### DISCUSSION THREAD

**Discussion by hsd0937** [NTNU-IHB/PythonFMU#199] [Q&A] [answered]
Hello,

I am currently using the pythonfmu library in a Python environment to generate FMU files. When I run my generated FMU file and observe the plot, I notice that the time (s) is displayed as 1.

I suspect this might be due to the Default Experiment settings not being configured properly.

Could you please provide me with the code to set the start time, stop time, and step size in the Default 
Experiment? (Or if there is another method to address this issue, I would appreciate your guidance.)

Thank you for your assistance.

![Default Experiment](https://github.com/NTNU-IHB/PythonFMU/assets/36781692/fdaffbd8-c277-42e4-b821-99e8391b3527)

**Comment by markaren** [comment, marked as answer]
In your constructor, you would have something like:

```python
self.default_experiment = DefaultExperiment(start_time=0)
```

However, the issue is probably not related to the default experiment as it is just static information written in the XML file. You may also edit the XML directly to change these values (which may or may not be used for anything depending on the importer)
